card/spec/lib/card/format/render_spec.rb

```diff
@@ -0,0 +1,19 @@
+describe Card::Format::Render do
+  describe "view cache" do
+    before { Cardio.config.view_cache = true }
+
+    let(:cache_key) do
+      "a-Card::Format::HtmlFormat-normal-home_view:content;"\
+      "nest_name:A;nest_syntax:A|content;view:contentcontent:show"
+    end
+
+    subject { Card::Cache[Card::View] }
+
+    it "can be changed with nest option" do
+      is_expected.to receive(:fetch).with cache_key
+      render_content "{{A|content}}"
+      is_expected.not_to receive(:fetch)
+      render_content "{{A|cache:never}}"
```

Comment:
1. more precisely, it's a general tool to fix (individually) this kind of bug, but not the general solution.
2. When I started that I thought the +companies nest was in a structure card. I didn't notice that it was overridden with a code structure. I'm fine with making it a code and not a wagneer option
3. I'm not sure what makes more sense. The value of this feature is that you can turn off caching for views you don't have a code rule for. Since there is no way to address subviews with that the question is if it's enough to have only the possibility to disable caching for the home view